dialoguekit/core/feedback.py

```diff
@@ -0,0 +1,18 @@
+"""Interface representing user's feedback."""
```

Comment:
Nit: 
```
"""Interface representing user feedback.

Currently only binary feedback is supported on the utterance level. Later, it might be extended to graded feedback as well as with conversation-level feedback.
"""
```